service/service.go

```diff
@@ -91,7 +100,16 @@ func (svc *Service) Close(ctx context.Context) error {
 			hasShutdownError = true
 		}
 
+		// close mongoDB
+		if svc.ServiceList.MongoDB {
+			if err := svc.MongoDB.Close(ctx); err != nil {
+				log.Event(ctx, "error closing mongoDB", log.Error(err), log.ERROR)
+				hasShutdownError = true
+			}
+		}
+
 		// ADD CODE HERE: Close other dependencies, in the expected order
```

Comment:
I think this comment can be removed now that we are already closing dependencies

Reply:
Same as above